### ISSUE TITLE
Issue with Garden Saw/Suture Needle

### ISSUE THREAD

**Issue by ScrooblesMcDoodle** [ZioPao/The-Only-Cure#68]
Whenever I hold both the Garden Saw and the Suture Needle in my main inventory, I get this error:

```
LOG  : General     , 1715520966617> 2,252,367,152>
----------------------------------------- STACK TRACE -----------------------------------------
function: GetStitchesConsumableItem -- file: CutLimbInteractionHandler.lua line # 42 | MOD: The Only Cure
function: AddInventoryAmputationMenu -- file: CutLimbInteractionHandler.lua line # 171 | MOD: The Only Cure
Callframe at: se.krka.kahlua.integration.expose.MultiLuaJavaInvoker@c1587d71
function: createMenu -- file: ISInventoryPaneContextMenu.lua line # 787 | Vanilla
function: onRightMouseUp -- file: ISInventoryPane.lua line # 1444 | Vanilla
```

This only happens when they are both in my main inventory. If I put the Suture Needle in my backpack this error doesn't occur.

**Comment by ScrooblesMcDoodle** [comment]
As an addition to this, if both the saw and suture needle are in your main inventory it doesn't let me saw my arm. The context prompt just isn't there. As soon as I move the needle into my backpack however, the option comes back.

**Comment by ZioPao** [comment]
Same issue as #85, fixed in v2.0.15
